app/index.html

```diff
@@ -0,0 +1,99 @@
+<!doctype html>
+<html ng-app="diplomacy">
+  <head>
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+	<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
+	<script type="text/javascript" src="app.js"></script>
+  </head>
+  <body>
+	<div class='col-md-6' ng-controller="MatchesController as matchesCtrl">
```

Comment:
You started using quotation marks around attribute values, then switched to apostrophes.  Be consistent above all else, but you probably want to use quotations.  While both are accepted, I think quotations are pretty well the standard.